eclass/dist-kernel-utils.eclass

```diff
@@ -19,6 +19,14 @@
 # If set to a non-null value, inherits secureboot.eclass
 # and allows signing of generated kernel images.
 
+# @ECLASS_VARIABLE: KERNEL_EFI_ZBOOT
+# @PRE_INHERIT
```

Comment:
Why is it pre-inherit? oO

Reply:
I guess this is not technically required. It should definitely be set in global scope though because lots of things are calling `dist-kernel_get_image_path`.